### PR TITLE
AIRNEIS-106: Frontend translations + doc

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
# DEPENDENCIES (optional)

# DONE 

Made translation for frontend  

# TEST

- Do a `make dev` and go to `/`

### CODE REVIEW

**Review by KyDev25 — APPROVED** [review]
I've tested it and it works very well, very good job. As far as the doc is concerned, everything works for me, which is understandable.

File: front/public/locales/en/common.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Hello": "Hello",
+  "homePage": {
+    "wip": "WIP come back later!",
+    "linkExample": "Exemple de lien"
```
Comment (cedricgautier) [inline front/public/locales/en/common.json]: Can you please fix the translation, if not mistaken it's supposed to be in English.

File: front/public/locales/fr/common.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "hello": "Bonjour",
+  "homePage": {
+    "wip": "Travail en cours, revenez plus tard !",
+    "linkExample": "Exemple de lien"
+  },
+  "header": {
+    "selectALanguage": "Séléctionnez une langue",
```
Comment (cedricgautier) [inline front/public/locales/fr/common.json]:
```suggestion
    "selectLanguage": "Séléctionnez une langue",
```

**Review by cedricgautier — CHANGES_REQUESTED** [review]

**Review by cedricgautier — APPROVED** [review]

File: front/src/components/Header.jsx [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
+    push("", "", { locale })
+  }
+
+  return (
+    <header className="bg-secondary text-primary px-4 flex justify-between items-center">
+      <h1 className="text-4xl py-6">ÀIRNEIS</h1>
       <nav>
         <ul className="flex flex-row items-center gap-5">
+          <li>
+            <select className="bg-secondary " onInput={({ target: { value } }) => changeLocale(value)}>
```
Comment (avetisk) [inline front/src/components/Header.jsx:19]: Handlers should be declared outside of JSX.

**Pull request merged**